aiohttp/client_reqrep.py

```diff
@@ -860,6 +861,13 @@ async def start(self, connection: "Connection") -> "ClientResponse":
                     set_result(self._continue, True)
                     self._continue = None
 
+        # certificate
+        is_tls_transport = (
+            self._connection is not None and self._connection.transport is not None
+        )
```

Comment:
Ah, but this variable trips it up. mypy can't infer that the next line based on the connection variable. Just stick to putting this logic in the `if` without a variable.